core/templates/pages/preferences-page/preferences-page.component.html

```diff
@@ -42,7 +42,7 @@
       <div class="form-group row">
         <label for="label-target-user-bio" class="col-lg-4 col-md-4 col-sm-4" translate="I18N_PREFERENCES_BIO"></label>
         <div class="col-lg-8 col-md-8 col-sm-8">
-          <textarea id="label-target-user-bio" class="oppia-bio-border protractor-test-user-bio" ng-model="$ctrl.userBio" ng-blur="$ctrl.saveUserBio($ctrl.userBio)" rows="5" maxlength="2000"></textarea>
+          <textarea id="label-target-user-bio" class="oppia-bio-border protractor-test-user-bio" ng-model="$ctrl.userBio" ng-blur="$ctrl.saveUserBio($ctrl.userBio)" ng-change="$ctrl.checkBioChanged()" rows="5" maxlength="2000"></textarea>
```

Comment:
While you're here let's improve the readability
```suggestion
          <textarea id="label-target-user-bio" 
                    class="oppia-bio-border protractor-test-user-bio" 
                    ng-model="$ctrl.userBio" 
                    ng-blur="$ctrl.saveUserBio($ctrl.userBio)" 
                    ng-change="$ctrl.checkBioChanged()" 
                    rows="5" 
                    maxlength="2000">
          </textarea>
```

Reply:
@vojtechjelinek Should I do this for all `<input>` tags?

Reply:
Up to you, when I do some changes to HTML file I will usually only fix the stuff I touch and leave the rest.

Reply:
Up to you, when I do some changes to HTML file I will usually only fix the stuff I touch and leave the rest.

Reply:
DONE